contracts/compliance/modular/modules/MaxBalanceModule.sol

```diff
@@ -107,6 +107,8 @@ contract MaxBalanceModule is AbstractModuleUpgradeable {
 
     error TokenAlreadyBound(address _compliance);
 
+    error IdentityNotFound();
```

Comment:
i would add the wallet address for which identity was not found as a parameter here, would help to debug in case of batch transfers and know which sub-transfer failed